src/goto-instrument/contracts/utils.h

```diff
@@ -116,6 +116,8 @@ const symbolt &new_tmp_symbol(
   const typet &type,
   const source_locationt &location,
   const irep_idt &mode,
-  symbol_table_baset &symtab);
+  symbol_table_baset &symtab,
+  std::string suffix = "tmp_cc");
```

Comment:
This new parameter needs a doxygen comment above

Reply:
fixed